docs/en/latest/plugins/kafka-logger.md

```diff
@@ -1,5 +1,11 @@
 ---
 title: kafka-logger
+keywords:
+  - APISIX
+  - Plugin
+  - Kafka Logger
+  - kafka-logger
```

Comment:
Hi @navendu-pottekkat, for the keyword `Kafka Logger` and `kafka-logger`, after thinking, I would prefer only to keep one of them, like `Kafka Logger.` This is because when we search keywords on Google, both are the same for search engines in my experience (I don't have direct rules to prove this).

Also, how about adding `API Gateway` and `Kafka Log` here? bcoz there will have some query like `api gateway kafka log`